engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/deployment/DeploymentPreparer.scala

```diff
@@ -84,6 +85,7 @@ class DeploymentPreparer(config: K8sDeploymentManagerConfig) extends LazyLogging
     def modifyRuntimeContainer(value: Container): Container = {
       val containerLens = GenLens[Container](_.env).modify(_ ++ runtimeContainer.env) andThen
         GenLens[Container](_.volumeMounts).modify(_ ++ runtimeContainer.volumeMounts) andThen
+        GenLens[Container](_.ports).modify(_ ++ runtimeContainer.ports) andThen
```

Comment:
add test to DeploymentPrepererSpec

Reply:
actually this change was not needed. deleted.